src/pysdmx/model/message.py

```diff
@@ -0,0 +1,35 @@
+"""Message module.
+
+This module contains the enumeration for the different types of messages that
+can be written.
+"""
+
+from datetime import datetime
+from enum import Enum
+from typing import Optional
+
+from msgspec import Struct
+
+
+class ActionType(Enum):
+    """ActionType enumeration.
+
+    Enumeration that withholds the Action type for writing purposes.
+    """
+
+    Append = "append"
+    Replace = "replace"
+    Delete = "delete"
+    Information = "information"
+
+
+class Header(Struct, frozen=True, kw_only=True):
+    """Header for the SDMX messages."""
+
+    id: str = "test"
```

Comment:
Sorry for not spotting this earlier: Instead of hardcoding the ID, maybe with could use the Python uuid module, and its uuid4 function? After all, the ID is expected to be unique for each message so, that would be a good default unless the caller sets the desired ID?

Reply:
Same answer as with the prepared default value. I like it, but need to find a way to properly test it. I will have another look at it.

Reply:
Changed to str(uuid4()) on latest commit.